Gems/Atom/Feature/Common/Assets/Shaders/Silhouette/SilhouetteGather.shader

```diff
@@ -0,0 +1,66 @@
+{
+    "Source" : "SilhouetteGather.azsl",
+    "DepthStencilState" : {
+        "Depth": 
+        {
+            "Enable": true, 
+            "WriteMask" : "Zero",   // Avoid writing the depth
+            "CompareFunc" : "Less"
+        },
+        "Stencil" :
+        {
+            "Enable" : true,
+            "ReadMask" : "0x4",
+            "WriteMask" : "0x0",
```

Comment:
I do not understand why the mask is zero, there is Replace operations below. Probably I missed something.